bin/morning_seal.sh

```diff
@@ -1,18 +1,18 @@
 #!/bin/bash
 
 teams=(
-  modelling-services
+  content-api
```

Comment:
If you get a quick chance to remove this as the team no longer exists that'd be super 👍